src/main/java/com/faforever/client/rankedmatch/MatchmakerMessage.java

```diff
@@ -11,26 +11,42 @@
 
   public static class MatchmakerQueue {
 
-    private String queueName;
+    private QueueName queueName;
+    @SerializedName("queue_pop_time")
```

Comment:
Remove this it is not needed due to FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES set with Gson

Reply:
kk